### PR TITLE
Port scripts to Python3

### PR DESCRIPTION
**Pull request by chuan-wang**
Scripts also back support Python 2. A validation will be started for the changes.

### CODE REVIEW

**Comment by chuan-wang** [comment]
Validation 21_10 has been approved.

File: scripts/quantit_generate_driver_file.py [hunk 1]
```diff
@@ -47,22 +47,22 @@ def make_location_dict(self, io_filtered):
...
+            print(location_dict[key], file=f)
         f.close()
 
 def main(lims, pid, drivf ,epp_logger):
     process = Process(lims,id = pid)
     QiT = QuantitDriverFile(process, drivf)
     io = process.input_output_maps
-    io_filtered = filter(lambda (x,y): y['output-generation-type']=='PerInput', io)
-    io_filtered = filter(lambda (x,y): y['output-type']=='ResultFile', io_filtered)
+    io_filtered = [x_y for x_y in io if x_y[1]['output-generation-type']=='PerInput']
```
Comment (aanil) [inline scripts/quantit_generate_driver_file.py:64]: Is there a reason the variable is named `x_y` and `x_y1` instead of just `x` or `y`?

Comment (chuan-wang) [reply scripts/quantit_generate_driver_file.py:64]: No, this is the suggested change by `2to3`

Comment (aanil) [reply scripts/quantit_generate_driver_file.py]: 👍 Ok!

**Review by aanil — APPROVED** [review]
The changes look ok to me!

**Pull request merged**